client/src/components/atoms/Spinner.module.scss

```diff
@@ -39,7 +39,7 @@
         height: $font-size-small;
         margin-top: -($font-size-small);
         margin-left: -($font-size-small / 2);
-        border-width: .1rem;
+        border-width: 0.1rem;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)